src/datePicker/index.tsx

```diff
@@ -1,132 +1,194 @@
-import React, { Component } from 'react'
+import React, { PureComponent } from 'react'
 import cc from 'classcat'
-import moment from 'moment'
-import 'react-dates/initialize'
-import { DayPickerSingleDateController, isInclusivelyAfterDay } from 'react-dates'
-
-import {
-  HORIZONTAL_ORIENTATION,
-  VERTICAL_ORIENTATION,
-  VERTICAL_SCROLLABLE,
-} from 'react-dates/constants'
+import DayPicker, { DayModifiers, NavbarElementProps, CaptionElementProps } from 'react-day-picker'
 
+import prefix from '_utils'
 import { color } from '_utils/branding'
 import ArrowIcon from 'icon/arrowIcon'
-import style from './style'
-import theme from './theme'
+import Button from 'button'
+
+import style from 'datePicker/style'
+
+const [BASE_CLASSNAME] = prefix({ datepicker: true })
 
-const navPrev = <ArrowIcon iconColor={color.primary} />
-const navNext = <ArrowIcon right iconColor={color.primary} />
+const defaultWeekdaysLong = [0, 1, 2, 3, 4, 5, 6].map(weekday =>
+  DayPicker.LocaleUtils.formatWeekdayLong(weekday),
```

Comment:
Is it using Intl to translate by default? If so, is it using the browser locale?

Reply:
It's not using Intl, ReactDayPicker has basic english default settings (with months and weekdays definitions).
So what happens is that our DatePicker is english by default, and you can override everything with the given props.

Reply:
nit: Maybe add some doc in the storybook to explain how to use Intl for this.

Reply:
Done, that's a good point even though the story code provides an example, at least it is visible at first glance.